tox.ini

```diff
@@ -0,0 +1,17 @@
+[tox]
+envlist = py27
+project = yelp
+
+[testenv]
+deps = -rrequirements-dev.txt
```

Comment:
This covers the dev requirements; I expect we'll get `httplib` & `oauth2` from `setup.py`, reasoning through how that'll happen.